src/prompt.c

```diff
@@ -538,8 +539,11 @@ prompt_histfile(void)
 			die("Failed to expand $HOME");
 	} else if (!string_format(path, "%s/tig/history", xdg_data_home))
 		die("Failed to expand $XDG_DATA_HOME");
-	else
-		mkdir(dirname(path), 0777);
+	else {
+		static char path_copy[SIZEOF_STR] = "";
```

Comment:
`path_copy` won't be used outside of prompt_histfile(), so we don't need to make it static. 
```
char path_copy[SIZEOF_STR];
```
should be enough here, don't you think ?

Reply:
That looks better. I don't remember why I added `static` :joy: